GLMakie/src/screen.jl

```diff
@@ -590,24 +597,21 @@ function closeall()
     return
 end
 
-function resize_native!(window::GLFW.Window, resolution...)
+function resize_native!(window::GLFW.Window, w, h)
     if isopen(window)
         ShaderAbstractions.switch_context!(window)
-        oldsize = windowsize(window)
-        retina_scale = retina_scaling_factor(window)
-        w, h = resolution ./ retina_scale
-        if oldsize == (w, h)
-            return
-        end
-        GLFW.SetWindowSize(window, round(Int, w), round(Int, h))
+        windowsize(window) == (w, h) && return
+        GLFW.SetWindowSize(window, w, h)
     end
+    return
 end
```

Comment:
I thought the reduction in window size was happening here...

Looking at GLFW docs, there seems to be a retina window hint which we don't have in GLFW.jl

> `GLFW_COCOA_RETINA_FRAMEBUFFER` specifies whether to use full resolution framebuffers on Retina displays. Possible values are `GLFW_TRUE` and `GLFW_FALSE`. This is ignored on other platforms.

https://www.glfw.org/docs/latest/window_guide.html

and there is also a compile flag

> `GLFW_USE_RETINA` determines whether windows will use the full resolution of Retina displays.

https://www.glfw.org/docs/3.1/compile.html

Maybe we also need some changes in GLFW?

Reply:
I think retina is already used correctly, you specify a window size of 400x300 and you get a frame buffer size of 800x600. It's just that the size of everything should be doubled in pixels so that it looks normal.

You can use the retina flag to disable that behavior, then you'd get a pixelated window though, I think.